tests/sentry/tasks/test_performance_detection.py

```diff
@@ -0,0 +1,250 @@
+from datetime import timedelta
+from unittest.mock import Mock, call, patch
+
+from sentry.tasks.performance_detection import _detect_performance_issue, detect_performance_issue
+from sentry.testutils import TestCase
```

Comment:
if you don't need db access, use `unittest.TestCase`, it will speed up your tests.